src/astra_driver.cpp

```diff
@@ -188,17 +188,17 @@ void AstraDriver::advertiseROSTopics()
     //image_transport::SubscriberStatusCallback itssc = boost::bind(&AstraDriver::colorConnectCb, this);
     //ros::SubscriberStatusCallback rssc = boost::bind(&AstraDriver::colorConnectCb, this);
     //pub_color_ = color_it.advertiseCamera("image", 1, itssc, itssc, rssc, rssc);
-    pub_color_ = nh_->create_publisher<sensor_msgs::msg::Image>("image", custom_camera_qos_profile);
-    this->colorConnectCb();
+    //pub_color_ = nh_->create_publisher<sensor_msgs::msg::Image>("image", custom_camera_qos_profile);
+    //this->colorConnectCb();
```

Comment:
Ah, OK, I wasn't sure if anyone was using it.  There are two things here:

- I may be able to get away without disabling these; it just reduces CPU load for me.
- I'm not opposed to making them parameters, but I haven't been able to get parameters to work really, at all.  `ros2param list` always returns an empty list for me.  Do parameters work for anyone else?